controllers/actions.summerwind.net/horizontal_runner_autoscaler_webhook.go

```diff
@@ -115,7 +115,7 @@ func (autoscaler *HorizontalRunnerAutoscalerGitHubWebhook) Handle(w http.Respons
 	}()
 
 	// respond ok to GET / e.g. for health check
-	if r.Method == http.MethodGet {
+	if ToUpper(r.Method) == http.MethodGet {
```

Comment:
```suggestion
	if strings.ToUpper(r.Method) == http.MethodGet {
```

Something is wrong with this 😉